tests/test_sft_trainer.py

```diff
@@ -695,6 +717,44 @@ def test_run_causallm_lora_and_inference(request, target_modules, expected):
         assert len(output_inference) > 0
         assert "Simply put, the theory of relativity states that" in output_inference
 
```

Comment:
@kgreenewald Thanks for adding the unit test. You would need to decorate it with a condition to run only when alora package is locally available. 
1. Define a utility function. example. https://github.com/foundation-model-stack/fms-hf-tuning/blob/b3744ca0d5031c6e3a406190e0de8bdbe0a6900f/tuning/utils/import_utils.py#L22
2. Decore the unit test to run only if package available. example. https://github.com/foundation-model-stack/fms-hf-tuning/blob/b3744ca0d5031c6e3a406190e0de8bdbe0a6900f/tests/test_sft_trainer.py#L1453-L1456

Reply:
Apart from this, I vouch unit tests pass at my end as well, thank you!